### ISSUE TITLE
`formatter` -fix removes `fmt.Sprintf` but doesn't suffix assertion method with `f`

### ISSUE THREAD

**Issue by mmorel-35** [Antonboom/testifylint#177]
With master version of testifylint, with (Example from https://github.com/kubernetes/kubernetes/pull/127649/files)

```go 
assert.Error(t, err, fmt.Sprintf("Profile %s should not be valid", test.profile))
```

The suggestion is

```go 
assert.Error(t, err, "Profile %s should not be valid", test.profile)
```

But I expect 

```go 
assert.Errorf(t, err, "Profile %s should not be valid", test.profile)
```

**Comment by Antonboom** [comment]
@mmorel-35 hi!

did you use `--formatter.require-f-funcs` flag?

**Comment by Antonboom** [comment]
Your expectations is invalid.
Please, read the checker doc carefully, including historical reference.

By Go convention f-suffix is triggered by any message, without arguments too.
Look at `fmt.Sprint` and `fmt.Sprintf` for example,
